stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -319,15 +319,13 @@ public struct AsyncStream<Element> {
   /// the `unfolding` parameter label.
   ///
   ///     let stream = AsyncStream<Int> {
-  ///             await Task.sleep(1 * 1_000_000_000)
-  ///             return Int.random(in: 1...10)
-  ///         }
-  ///         onCancel: { @Sendable () in print ("Canceled.") }
-  ///     )
```

Comment:
Nice job catching this stray `)` that shouldn't be there.

Reply:
Thanks! 😄 
Note that I also edited the word "Canceled." → `"Cancelled."` (double `l`)

Reply:
@amartini51 Thanks for following up!
The [Apple Style Guide - C](https://help.apple.com/applestyleguide/#/apsgb744e4a3#apdb51b39ece6204) definitely says that we should use one `l` for `"canceled"`.

On [lines 121-122](https://github.com/apple/swift/blob/80ef80fef9a891c39a7b9a119041b5c9f0e01e21/stdlib/public/Concurrency/AsyncStream.swift#L121-L122) in the very same file, however, there is `case cancelled` in use.

```swift
/// The stream finished as a result of cancellation.
case cancelled
```

Actually, "cancelled" is being used much more than "canceled" in overall Swift repo. 🧐
Would it be better to just revert this word for now? How do you think?

Reply:
Let's revert this change please, so we're not making the problem worse.  Someone (possibly me or @invalidname) can take another pass over the file to correct the other "cancelled" spellings in a different PR.

Reply:
Regarding lines 121-121, `case cancelled` is an example of the Cocoa naming guidelines I mentioned.  You can see this style difference in API reference for `Task` and and related types — symbols with "cancelled" in its name have "canceled" in their docs.

Both style guides use the spelling "cancellation".

Reply:
Ohh, now I see. Documentation follows "canceled", while the API naming follows "cancelled".
Sorry for additional revert commit, and thank you so much for being part of this.

Reply:
Reading my original post, I could have been clearer that there were two style guides involved here, one for code and one for docs. Sorry about that. Thanks for the fixes in this PR!